src/configure.rs

```diff
@@ -40,11 +40,16 @@ fn get_editor_internal(visual: Option<OsString>, editor: Option<OsString>) -> Os
 }
 
 fn get_config_path() -> OsString {
-    dirs::home_dir()
-        .expect("Couldn't find home directory")
-        .join(".config/starship.toml")
-        .as_os_str()
-        .to_owned()
+    let config_path = env::var_os("STARSHIP_CONFIG").unwrap_or_else(|| "".into());
+    if config_path.is_empty() {
```

Comment:
Personally, I'd probably write

```rust
if !config_path.is_empty() {
	config_path
}

dirs::home_dir()
	.expect("couldn't find home directory")
	.join(".config/starship.toml")
	.as_os_str()
	.to_owned()
```

... skipping the `else` block, but that's up to you of course.

Reply:
I think the if/else is clearer here (as opposed to get_editor_internal, because there it's two after another)

Reply:
I might be a bit Go-influenced here. 😃

Reply:
Now that you mention it, that *really* looks like Go, yeah :P